cleanlab_studio/studio/studio.py

```diff
@@ -72,6 +72,8 @@ def download_cleanlab_columns(
                 rows_df = rows_df.drop("action")
             else:
                 rows_df.drop("action", inplace=True, axis=1)
+        if "cleanlab_row_ID" in rows_df.columns:
+            rows_df.sort_values(by="cleanlab_row_ID")
```

Comment:
don't you need a `inplace=True` here?

Reply:
How odd. You're right and yet my testing worked. Must have been a fluke I suppose?